monero-outreach-docs/translations/es/storyboard/randomX_es.md

```diff
@@ -0,0 +1,35 @@
+# RandomX
+*06/05/19*  
+_**Monero y Arweave a validar el algoritmo de Prueba-de-Trabajo.**_  
+ 
+¿Qué año es? Parece que solo fue ayer cuando decían que una moneda privada no podía reducir su tamaño de almacenamiento y que los ASICs ganarían al final. Pero aquí estamos, con los cascos espaciales puestos...reduciendo el tamaño del almacenamiento y hablando de RandomX. RandomX aún debe pasar por un proceso de prueba y autenticidad, pero es una parte emocionante del desarrollo de Monero. Esta guía da una descripción general de su naturaleza y plan, y, para que puedas aprender más. 
+
+## _¿Qué es RandomX?_  
+
+RandomX es un algoritmo de prueba-de-trabajo (PoW - siglas en inglés) nuevo que está programado para ser usado en la próxima actualización de Monero. RandomX está diseñado para ser resistente a los ASIC al utilizar códigos de ejecución aleatorios y técnicas de memoria exigentes para prevenir la dominación de la red por hardware de minería especializado. Siendo el proposito general de RandomX optimizar los CPUs, la distribución de la recompensa en la red será más descentralizada e igual para todos. 
+ 
+Howard Chu (hyc) estará hablando sobre RandomX en la conferencia de Monero, [Monero Konferenco](https://monerokon.com/), mientras tanto, visita el [repositorio en Github de RandomX](https://github.com/tevador/RandomX) para más detalles. Al momento de escribir este artículo, RandomX está empezando el proceso de ser auditado por [Trail Of Bits](https://www.trailofbits.com/), [X41](https://www.x41-dsec.de/), [Quarkslab](https://www.quarkslab.com/en/) y el [Grupo Kudelski](https://www.nagra.com/). RandomX está programado a lanzarse en la próxima actualización en la red de Monero.
+
+## _¿Qué va a cambiar?_ 
+
+Los ASICs serán los principales afectados por RandomX, y como será optimizado para los CPUs, los GPUs no tendrán el mismo incremento en el hashrate o en el poder de hashes. Referencias anteriores en las tarjetas de video [Nvidia (CUDA)](https://github.com/SChernykh/RandomX_CUDA) revelan un incremento del hashrate entre un 100% y un 150%, y se espera que se mejore con mayor optimización. Al ser RandomX exigente en memoria, se espera que los botnets y el malware en la minería se reduzca al ser el consumo de la memoria fácilmente detectado por los administradores. Una reducción en general del total del hashrate en la red incrementará la recompensa a los mineros legítimos, sin importar si minan con CPUs o GPUs. 
```

Comment:
According to the Spanish terminology, hashrate = índice de hash_. Sounds weird to me since hashing power = _poder de procesamiento_, we can improve that translation and put another one more accurate like:

+ _Razón de procesamiento._
+ _Índice de procesamiento._
+ _Tasa de procesamiento._

Reply:
Those are good options. We can add them to the Spanish terminology to have a wider definition.

Yes, it seems I forgot that sentence. :see_no_evil:

Reply:
In this case I will use Índice de procesamiento because it fits the definition. The word "tasa" is commonly used for measuring on a relationship to two different measures of magnitude. I'm not very clear how the hashrate is measured, so I will go with "índice" with this one. :)

Reply:
changed and added :)